tests/src/JIT/config/benchmark/benchmark.csproj

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
-    <NugetTargetMoniker>.NETStandard,Version=v1.4</NugetTargetMoniker>
-    <NugetTargetMonikerShort>netstandard1.4</NugetTargetMonikerShort>
+    <NugetTargetMoniker>.NETStandard,Version=v1.6</NugetTargetMoniker>
+    <NugetTargetMonikerShort>netstandard1.6</NugetTargetMonikerShort>
```

Comment:
What is the switch from 1.4 to 1.6 for?  We probably need to check with the JIT folks if we're going to make this change to their tests.

Reply:
I'd assumed that since we moved TraceEvent to 1.6, that this project wouldn't build unless also bumped to 1.6. That was incorrect